src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -64,11 +73,20 @@ public class SnowflakeSinkConnector extends SinkConnector {
   // Using setupComplete to synchronize
   private boolean setupComplete;
 
+  public static IngestSdkProvider ingestSdkProvider;
+
   /** No-Arg constructor. Required by Kafka Connect framework */
   public SnowflakeSinkConnector() {
     setupComplete = false;
   }
 
+  @Override
+  public void initialize(ConnectorContext ctx) {
```

Comment:
initialize() is the first method called on a clean connector

Reply:
since java is pass by value, not reference, there is no point in creating the object here - remove this function